bundles/org.openhab.binding.siemensrds/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -149,7 +149,7 @@
 	</channel-type>
 
 	<channel-type id="roomHumidity">
-		<item-type>Number:Dimensionless</item-type>
```

Comment:
See #16616

Reply:
In #16616 I also removed the “r.H.” from state description pattern. If you do that here in this PR then I can close that one.

Reply:
Although it's just one line, I think it would be cleaner to keep them separate, since that is actually a bugfix. I don't think there will be any conflict since we both changed the same line. You can remove the unit hint from your PR now, because it's already covered here. Then your PR can be merged already as it won't have any dependencies. Or you can rebase if my PR is merged first, which will give the same result. Or you can leave it as is, but then this PR must also be merged first.

Reply:
> You can remove the unit hint from your PR now, because it's already covered here. Then your PR can be merged already as it won't have any dependencies.

Ok done.